include/boost/variant/recursive_wrapper.hpp

```diff
@@ -125,8 +126,8 @@ recursive_wrapper<T>::recursive_wrapper(const T& operand)
 
 #ifndef BOOST_NO_CXX11_RVALUE_REFERENCES 
 template <typename T>
-recursive_wrapper<T>::recursive_wrapper(recursive_wrapper&& operand)
-    : p_(new T( detail::variant::move(operand.get()) ))
```

Comment:
Could we have a macro to restore the old behavior? Just for some time, so that users could migrate simpler.

Reply:
I have added macro, though I am not sure about its name.